build-system/compile/compile-wrappers.js

```diff
@@ -34,6 +34,16 @@ exports.mainBinary =
   's.animation="none";' +
   's.WebkitAnimation="none;"},1000);throw e};';
 
+/**
+ * Wrapper that either registers the extension or schedules it for execution
+ * by the main binary
+ * @param {string} name
+ * @param {string} version
+ * @param {boolean} latest
+ * @param {boolean=} isModule
+ * @param {'high'=} loadPriority
```

Comment:
What is this `{'high'=}` type syntax?

Reply:
The `build-system` checker uses TypeScript backed by jsdoc. This matches string literal `type LoadPriority = 'high'`, which could also be a union like `'high' | 'low'`

Reply:
Should this not be `{!LoadPriority=} loadPriority` then?

Reply:
Added a special type. No need for `!` outside Closure.